src/components/governance/PageProposal.vue

```diff
@@ -55,17 +55,31 @@
             <dt>Submitted</dt>
             <dd>{{ submittedAgo }}</dd>
           </dl>
+
           <dl class="info_dl colored_dl">
             <dt>Proposal Status</dt>
-            <dd>
-              {{
-                proposal.proposal_status === `DepositPeriod`
-                  ? `Deposit period ends ${depositEndsIn}.`
-                  : `Voting started ${votingStartedAgo}.`
-              }}
+            <dd v-if="proposal.proposal_status === 'DepositPeriod'">
+              {{ `Deposit period ends ${depositEndsIn}.` }}
```

Comment:
The text content should probably be a computed value instead of copying the element n times.